### PR TITLE
B2 16 team member card

### PR DESCRIPTION
**Pull request by nelsW70**
Closes [B2-16](https://b2io.atlassian.net/browse/B2-16)

Closes [B2-17](https://b2io.atlassian.net/browse/B2-17)

Closes [B2-92](https://b2io.atlassian.net/browse/B2-92)

- cropped/added team images
- created TeamMemberCard component
- created TeamCardList component
- created TeamHero component
- created team.json
- created font variant for teamHero 
- fixed culture hero text by adding soft hyphen to longer text

**Notes** 

- after discussion, the decision was to stack the cards on mobile rather than a vertical scroll
- the zeplin files only contains mobile and 1600 so I took some poetic license in terms of design

**Next Steps**
- [ ] update readme with info for adding new team members
- [ ] get masked images for newest team members from bonfire and replace the existing ones
- [ ] replace TeamHero with reusable ImageContainer component once that task has been completed

[https://zpl.io/Vx4md6E](url)
<details>
<summary>mobile</summary>

Screen Shot 2021-05-14 at 3 25 59 PM

</details>

<details>
<summary>tablet</summary>

Screen Shot 2021-05-14 at 3 07 15 PM

</details>

[https://zpl.io/VkQDz5g](url)
<details>
<summary>1600</summary>

Screen Shot 2021-05-14 at 3 07 42 PM

</details>

<details>
<summary>2560</summary>

Screen Shot 2021-05-14 at 3 08 01 PM

</details>

### CODE REVIEW

File: src/components/TeamCardList.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        return (
+          <TeamMemberCard
+            alt={member.name}
+            key={index}
+            name={member.name}
+            position={member.position}
+            src={member.src}
+          />
+        );
+      })}
```
Comment (tborres) [inline src/components/TeamCardList.tsx:80]: ⛳ If you're interested in playing character count golf you can deconstruct and do an implicit return

```suggestion
      {team.map(({ name, position, src }, index) => (
        <TeamMemberCard
          alt={name}
          key={index}
          name={name}
          position={position}
          src={src}
        />
      ))}
```

File: src/components/TeamCardList.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      <div className="team-card-background"></div>
```
Comment (tborres) [inline src/components/TeamCardList.tsx:81]: 💬 If you don't expect content for the node then consider self closing.

```suggestion
      <div className="team-card-background" />
```

File: src/components/TeamCardList.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  columns: 1;
+  position: relative;
+
+  img {
+    z-index: 2;
+  }
+
+  ${atMinTablet} {
+    column-gap: ${spacing.md};
+    columns: 2;
```
Comment (tborres) [inline src/components/TeamCardList.tsx:19]: 💬 I think that the cards should switch from 1 to 2 columns earlier. Currently at a screen width of 690 it's still showing giant single column photos.

File: src/components/TeamCardList.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      top: -6.5%;
+      width: 100vw;
+      z-index: 0;
+    }
+  }
+`;
+
+export const TeamCardList: FC = () => {
+  return (
+    <TeamMasonry>
```
Comment (tborres) [inline src/components/TeamCardList.tsx:69]: ❓ I'm really interested to know why there is a reshuffle of the cards between `889` and `890`.

**Review by tborres — COMMENTED** [review]
This is looking really good, especially given the lack of direction we can take from the comps on this one. With the change to stacking at mobile rather than horizontally scrolling I assume that some of the visual differences at mobile are intentional, and I think that the current implementation looks good. Just in case the differences weren't intentional I made a couple of callouts. There are also some resizing flow considerations I think we'll want to make to ensure that it looks "appropriate" at all screen sizes.

Not a big deal, but the comps have the lingering team card at the end be on the left (populating from left to right) rather than what the PR is showing with it appearing on the right. Not significant, but it does change the visual content weighting for the site.

Review the content, make changes as you see fit, and then I'll take another swing through it.

File: src/components/TeamCardList.tsx (hunk 3 above)
Comment (nelsW70) [reply src/components/TeamCardList.tsx:19]: Yeah, that makes sense

File: src/components/TeamMemberCard.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    z-index: 2;
+  }
+
+  & p {
+    margin-top: ${spacing.xxxs};
+    padding-left: ${spacing.md};
+    position: relative;
+  }
+
+  & p::before {
```
[the remark this answers is not in the document]
Comment (nelsW70) [reply src/components/TeamMemberCard.tsx:56]: @tborres, that was intentional but I'll remove it on mobile

File: src/theme/typography.ts [hunk 6]
```diff
@@ -167,6 +167,26 @@ export const textVariants = {
     margin: 0,
     padding: 0,
   },
+  teamHero: {
```
[the remark this answers is not in the document]
Comment (nelsW70) [reply src/theme/typography.ts:170]: @tborres, that was intentional on the text. Seems better to just keep it consistent

File: src/components/TeamMemberCard.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (tborres) [reply src/components/TeamMemberCard.tsx:56]: If it was intentional then feel free to leave as is. I don't have a strong opinion either way.

File: src/components/TeamCardList.tsx (hunk 1 above)
Comment (nelsW70) [reply src/components/TeamCardList.tsx:80]: Thaks @tborres , I should have caught that and the destructuring too

File: src/components/TeamCardList.tsx (hunk 4 above)
Comment (nelsW70) [reply src/components/TeamCardList.tsx:69]: @tborres, been looking at it and I think it's a combination of the next image growing and text-wrapping that's causing it

**Review by tborres — APPROVED** [review]
🚀 Ship it! Be sure to create the follow-up tasks (especially the readme update on how to add/remove team members).

**Pull request merged**